tutorials/input_and_output/input_and_output.md

```diff
@@ -364,14 +368,14 @@ core.chemical.ElementSet: New element: Pt
 ...
 core.chemical: Reading patch file: /home/ssrb/Rosetta/main/database/chemical/residue_type_sets/fa_standard/patches/CtermProteinFull.txt
 ...
-core.pose.util: new fold tree FOLD_TREE  EDGE 1 92 -1  EDGE 1 93 1  EDGE 1 94 2  EDGE 1 95 3  EDGE 1 96 4  EDGE 1 97 5  EDGE 1 98 6  EDGE 1 99 7 
+core.pose.util: new fold tree FOLD_TREE  EDGE 1 92 -1  
 ...
 ```
 
-Now you see a bunch of information that previously did not appear. In the snippet above, we see, the residue types and element types that Rosetta recognizes (C associated with aromatic rings and Platinum), all the patches it can apply (patch to make a residue the C-terminal) and the revised [fold tree](../minimization/minimization.md#a-note-about-the-foldtree).
+Now you see a bunch of information that previously did not appear. In the snippet above, we see, the residue types and element types that Rosetta recognizes (C associated with aromatic rings and Platinum), all the patches it can apply (patch to make a residue the C-terminal, this may be slightly different as it depends on the file paths on your machine) and the revised [fold tree](https://docs.rosettacommons.org/demos/latest/tutorials/minimization/minimization).
```

Comment:
Note that I'd recommend using relative links when possible, within the same repo. That way if someone is hosting the repo on their own server, the links will stay internal and won't bounce out to the main RosettaCommons one. (There's issues currently with setting up a local server, but that's something we can hopefully re-enable in the future.)

If the single bracket syntax isn't working well, I'd recommend trying the double bracket syntax. (I think `[[fold tree|tutorials/minimization/minimization]]` should work.